rfcs/0054-support-levels-and-tiers.md

```diff
@@ -0,0 +1,171 @@
+---
+feature: support_levels_and_tiers
+start-date: 2019-10-03
+author: CrystalGamma
+co-authors: –
+related-issues: –
+---
+
+# Summary
+[summary]: #summary
+
+A scheme for expressing and managing support of packages and for platforms in nixpkgs.
+
+# Motivation
+[motivation]: #motivation
+
+This RFC is an offshoot of a comment in the discussion on #46, which has been stuck in discussion for months.
+Its goals are similar, but this RFC focuses on what support expectations are reasonable for each level/tier.
```

Comment:
I think there should be a self-contained summary of motivation that makes independent sense a year later, when this RFC is finally accepted and #46 is rejected as horribly overengineered.

Reply:
Makes sense. Something like "Currently there is no consensus in the nixpkgs maintainer community regarding how to deal with platform-specific code and changes, as well as few official statements and processes to represent platform support to users. As a growing distribution, the Nix community needs to establish a framework to determine platform support and decide what platform-specific packaging code is acceptable."?

Reply:
Yes, that sounds reasonable